debootstrap/state.jinja

```diff
@@ -119,7 +119,7 @@ debootstrap_{{ data.id }}_sources_list:
     - mode: 644
     - template: jinja
     - context:
-        data: {{ data }}
+        data: {{ data|tojson }}
```

Comment:
```suggestion
        data: {{ data|json }}
```

For more info, please refer back to this comment: https://github.com/saltstack-formulas/consul-formula/pull/40#issuecomment-472728517.

Reply:
👍 I was not aware of the json and mostly followed https://docs.saltstack.com/en/latest/topics/releases/2019.2.1.html

Thanks for the tip 😉